Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -477,6 +476,39 @@ public static void StartGoogleSession(string idToken, Action<LootLockerGoogleSes
                 onComplete(response);
             });
         }
+        
```

Comment:
Instead of duplicating the code, would it make sense to call the overload from the original with the default platform?

Reply:
Actually I just got what you meant, I made a `LootLockerGoogleSignInWithPlatformSessionRequest` class that just extends the base one (with `platform` field), there's only a few lines of boilerplate duplication but we have that everywhere

Reply:
Aight, if you don't like it then ignore the comment ^^